### PR TITLE
attach intl tel input without jquery conflicts

### PR DESCRIPTION
**Pull request by demenkov**
or please use
$this->view->registerJs("$('#$id').parents('form').on('submit', function() { $('#$id').val($('#$id').intlTelInput('getNumber')); });");

### CODE REVIEW

File: src/PhoneInput.php [hunk 1]
```diff
@@ -29,9 +29,27 @@ public function init()
...
         $jsOptions = $this->jsOptions ? Json::encode($this->jsOptions) : "";
-        $this->view->registerJs("$('#$id').intlTelInput($jsOptions);");
+        $jsInit = <<<JS
+(function ($) {
+    "use strict";
+    $('#$id').intlTelInput($jsOptions);
+})(jQuery);
+JS;
+        $this->view->registerJs($jsInit);
+        $js = <<<JS
```
Comment (rumours86) [inline src/PhoneInput.php]: move after if($this->hasModel()) {

**Comment by Borales** [comment]
@demenkov thanks!
do we need a changelog line for your fix?

**Comment by demenkov** [comment]
@Borales as you wish
I think it breaks nothing or adding nothing, it's just keep code more clean.

**Comment by Borales** [comment]
@demenkov agree, thank you once again!

**Pull request merged**